PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/function/runner/localrun/FunctionRunConfiguration.java

```diff
@@ -217,6 +225,9 @@ public void validate() throws ConfigurationException {
         if (!func.exists() || !func.isFile() || !func.getName().contains("func")) {
             throw new ConfigurationException(message("function.run.validate.invalidFuncPath"));
         }
+        if (getFuncPort() <= 0 || getFuncPort() >= 65535) {
+            throw new ConfigurationException(message("function.validate_run_configuration.invalidPort"));
```

Comment:
use `AzureToolkitRuntimeException` please.

Reply:
This is to follow the framework of https://github.com/microsoft/azure-tools-for-java/blob/develop/PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/common/AzureRunConfigurationBase.java#L43, which will be called in `SettingsEditor.applyEditorTo`, uses `ConfigurationException`.